examples/accordion/css/accordion.css

```diff
@@ -1,7 +1,7 @@
 .accordion {
   margin: 0;
   padding: 0;
-  border: 2px solid hsl(0, 0%, 52%);
+  border: 2px solid hsl(0deg 0% 52%);
```

Comment:
Should we change https://github.com/w3c/aria-practices/wiki/Code-Guide#syntax-1 to say to use spaces instead of commas in `rgb()`, `hsl()` et. al.?

Reply:
It can be autofixed to either "modern" (spaces) or "legacy" (commas)
https://stylelint.io/user-guide/rules/list/color-function-notation

Support seems pretty good, so I think we can go with "modern".
https://caniuse.com/mdn-css_types_color_rgb_function_accepts_alpha

Reply:
Yeah, I autofixed the bulk of this using whatever the default is by the preset

Reply:
I think we can merge this. Updating the wiki is tracked in https://github.com/w3c/aria-practices/issues/2110